src/modules/navigator/mission_block.cpp

```diff
@@ -251,15 +251,26 @@ MissionBlock::is_mission_item_reached()
 				}
 
 			} else {
+				// Determine altitude acceptance radius. By default, this is taken from the respective parameter.
+				// However, before a landing approach the acceptance radius needs to be tighter: Assume a 30% error
+				// w.r.t. the remaining descent altitude is OK, but enforce min/max values (e.g. min=3m to make
+				// sure that the waypoint can still be reached in case of wrong user input).
+				float alt_accept_rad = _navigator->get_altitude_acceptance_radius();
+				struct position_setpoint_s next_sp = _navigator->get_position_setpoint_triplet()->next;
```

Comment:
This is a copy (extra work), use a const reference instead. In c++ you also don't need the "struct".
`struct position_setpoint_s next_sp` -> `const position_setpoint_s& next_sp`

Reply:
Changed.